src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(Color color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    public double getSide() {
+        return side;
+    }
+
+    public void setSide(double side) {
+        this.side = side;
+    }
+
+    @Override
+    public double calculateArea() {
+        return side * side;
+    }
+
+    @Override
+    public void drawFigure() {
+        System.out.println("Figure: square, area: " + calculateArea() + " sq. units, "
```

Comment:
Don't use `+` for string concatenation inside `System.out.println()` statements. Consider using `String.format()` for better readability and performance.